packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -427,6 +427,14 @@ const convertEventType = (type: string): ChatMessageType => {
   }
 };
 
+/**
+ * Option bag to include when creating AzureCommunicationChatAdapter.
+ * @public
+ */
+export type ChatAdapterOptions = {
+  credential?: CommunicationTokenCredential;
```

Comment:
Do we want to lock to the ACS specific type, rather than just passing through the necessary function to get the authToken?